ShoppingList/src/test/java/edu/agh/eaiib/CommandParserTest.java

```diff
@@ -0,0 +1,41 @@
+package edu.agh.eaiib;
+
+import edu.agh.eaiib.model.Product;
+import edu.agh.eaiib.service.InMemoryProductListRepository;
+import edu.agh.eaiib.service.ProductListService;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.junit.Assert.assertEquals;
+
+public class CommandParserTest {
+
+    private CommandParser testObject;
```

Comment:
why not name it commandParser?

Reply:
I'm a fan of enhancing object that is tested.

Reply:
When I see `testObject` in test code I'm like: "Hmm... What class is actually tested here?" and go to declaration.